Storage.md

```diff
@@ -0,0 +1,298 @@
+        }
+    }
+});
+
+// color is in the container, but speed is not
+ItemsHolder.addItem("speed", {
+    valueDefault: 10,
+    hasElement: true,
+    element: {}
+});
+```
+
+### Display Changes
+
+If you want certain values to be represented differently on the page, make use of `displayChanges`.
+These changes are hardcoded values that replace specified values when element items are updated.
+
+```typescript
+let ItemsHolder = new ItemsHoldr({
+    displayChanges: {
+        Infinity: "INF";
+    }
+});
+
+ItemsHolder.addItem("limit", {
+    value: "4000",
+    hasElement: true,
+    element: {}
+});
+ItemsHolder.setItem("limit", "Infinity");
+```
+
+`hasDisplayChange` checks to see if a value has a recorded change.
+
+```typescript
+ItemsHolder.hasDisplayChange("Infinity");
+```
+
+...and `getdisplayChange` returns the entry to replace the value with.
+
+```typescript
+let newValue: string = ItemsHolder.getDisplayChange("Infinity");
+```
+
+### Container Visibility
+
+`hideContainer` hides the container from view.
+
+```typescript
+ItemsHolder.hideContainer();
+```
+
+...while `displayContainer` makes the container visible.
+
+```typescript
+ItemsHolder.displayContainer();
+```
+
+More can be read about ItemsHoldr on its [Readme](https://github.com/FullScreenShenanigans/ItemsHoldr/blob/master/README.md).
+
+# StateHoldr
+
+StateHoldr is a module for tracking changes of items in ItemsHolder in objects called collections.
+A collection is an object whose keys are item names in ItemsHolder and whose values are key-value pairs of of item attributes and values.
+These key-value pairs of attributes and values are referred to as changes.
+Changes are separate from the actual values for items in ItemsHoldr.
+
+## Prefix
+
+Like ItemsHoldr, StateHoldr has its own prefix property.
+The prefixes for StateHoldr and ItemsHoldr are not the same and do not affect each other.
+StateHoldr needs a prefix of its own as it saves its collections in ItemsHoldr.
+
+```typescript
+let StateHolder = new StateHoldr({
+    ItemsHolder: new ItemsHoldr(),
+    prefix: "StateHolder"
+});
+```
+
+## Collections
+
+StateHoldr keeps track of the collections its recorded.
+It is saved as `stateCollectionKeys` in ItemsHolder.
+StateHoldr always points to one collection, the current collection, at any time.
+
+`setCollection` sets the current collection.
+
+```typescript
+StateHolder.setCollection("newCollection", {
+    car: {
+        color: "red"
+    }
+});
+```
+
+`saveCollection` saves the current collection to ItemsHolder.
+
+```typescript
+StateHolder.saveCollection();
+```
+
+## Changes
+
+To add a change to the current collection, use `addChange`.
+`addChange` takes in the key of the item, an attribute of the item, and the value being stored.
+
+```typescript
+StateHolder.addChange("car", "color", "red");
+```
+
+If the name of the collection is known, whether or not its the current collection, a change can be added with `addCollectionChange`.
```

Comment:
Grammar for here and other places: its is not the same as it's. Check your usage.